### PR TITLE
fix: cc-0027 delete easy-in-out effect

### PR DESCRIPTION
**Pull request by Beryl-Zhou**

### CODE REVIEW

File: src/layouts/HeaderLayout.tsx [hunk 1]
```diff
@@ -18,11 +18,6 @@ const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
   const headName = `${title} | CourtCanva`;
   return (
     <motion.article
```
Comment (tatoMa) [inline src/layouts/HeaderLayout.tsx:20]: Yes, this is the correct place for removing the transition of our pages but do you think we still need an empty motion.article tag?

File: src/layouts/HeaderLayout.tsx [hunk 2]
```diff
@@ -17,14 +17,6 @@ const variants = {
 const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
   const headName = `${title} | CourtCanva`;
   return (
-    <motion.article
-      initial="hidden"
-      animate="enter"
-      exit="exit"
-      variants={variants}
```
Comment (kritomomo) [inline src/layouts/HeaderLayout.tsx:24]: image
Looks like we don't need variants now

**Review by kritomomo — APPROVED** [review]
LGTM, Remember squash your commit message before merge

**Review by tatoMa — APPROVED** [review]
LGTM now.

**Pull request merged**